src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,31 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
+Animal.alive = [];
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(victim) {
+    if (victim instanceof Herbivore && victim.hidden !== true) {
```

Comment:
let's use shortcuts for boolean
```suggestion
    if (victim instanceof Herbivore && victim.hidden !== true) {
```
```suggestion
    if (victim instanceof Herbivore && !victim.hidden) {
```